hera_cal/abscal_funcs.py

```diff
@@ -1055,56 +1063,83 @@ def interp2d_vis(model, model_lsts, model_freqs, data_lsts, data_freqs,
 
     data_freqs : 1D array of the data freq axis, dtype=float, shape=(Nfreqs,)
 
-    kind : kind of interpolation method, type=str, options=['linear', 'cubic', ...]
-        see scipy.interpolate.interp2d for details
+    flags : type=DataContainer, dictionary containing model flags. Can also contain model wgts and
+                                will convert appropriately.
+
+    medfilt_flagged : type=bool, if True, run a median filter on flagged model pixels
 
-    fill_value : values to put for interpolation points outside training set
-        if None, values are extrapolated
+    medfilt_window : type=tuple, window for median filter across the (time, freq) axes.
 
-    zero_tol : for amplitudes lower than this tolerance, set real and imag components to zero
+    kx : type=int, type of spline to fit to freq axis of data. See scipy.interpolate.RectBivariateSpline
 
-    bounds_error : type=boolean, if True, raise ValueError when extrapolating. If False, extrapolate.
+    ky : type=int, type of spline to fit to time axis of data. See scipy.interpolate.RectBivariateSpline
 
-    flag_extrapolate : flag extrapolated data if True
+    s : type=int, smoothing factor for interpolation. Default is no smoothing. See scipy.interpolate.RectBivariateSpline
+
+    flag_extrapolate : type=bool, flag extrapolated data if True
```

Comment:
What happens if you have both medfilt_flagged and flag_extrapolate as true?

Reply:
`medfilt_flagged` will replace flagged data with a median filter centered at the flagged data point. That happens for all flagged data regardless of where it is in LST and frequency. 

`flag_extrapolate` will flag the boundary of the lst grid if it isn't nested between two data lst points

Reply:
Ok, I see. I was confused.